README.md

```diff
@@ -144,13 +144,42 @@ Commands
 
 | **Command**   | **Description**
 |:--------------|:---------------------------------------------------------------
+| `help`        | Shows the help dialog
 | `build`       | Invokes `make`/`msbuild.exe` and builds the native addon
 | `clean`       | Removes the `build` directory if it exists
 | `configure`   | Generates project build files for the current platform
 | `rebuild`     | Runs `clean`, `configure` and `build` all in a row
-| `install`     | Installs node development header files for the given version
-| `list`        | Lists the currently installed node development file versions
-| `remove`      | Removes the node development header files for the given version
+| `install`     | Installs node header files for the given version
+| `list`        | Lists the currently installed node header versions
+| `remove`      | Removes the node header files for the given version
+
+
+Command Options
+--------
+
+`node-gyp` accepts the following command options:
+
+| **Command**                       | **Description**
+|:----------------------------------|:------------------------------------------
+| `-j n`, `--jobs n`                | Run make in parallel
+| `--silly`, `--loglevel=silly`     | Log all progress to console
+| `--verbose`, `--loglevel=verbose` | Log most progress to console
+| `--silent`, `--loglevel=silent`   | Don't log anything to console
+| `debug`, `--debug`                | Make Debug build (default=Release)
+| `--release`, `--no-debug`         | Make Release build
+| `-C $dir`, `--directory=$dir`     | Run command in different directory
+| `--make=$make`                    | Override make command (e.g. gmake)
+| `--thin=$thin`                    | Enable thin static libraries
```

Comment:
Maybe write this as `--thin=yes`?  Currently the only requirement is that it needs to be a non-empty string.